package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "uploadcare-tinymce",
+  "version": "2.1.1",
+  "widgetVersion": "3.x",
+  "description": "File Uploader by Uploadcare. The plugin for TinyMCE to work with Uploadcare Widget.",
```

Comment:
File Uploader by Uploadcare, a plugin providing TinyMCE users to upload media via Uploadcare Widget.